### DISCUSSION TITLE
Ship a template to create templates

### DISCUSSION THREAD

**Discussion by yajo** [copier-org/copier#1011] [Ideas]
Quoting from https://github.com/copier-org/copier/issues/983#issue-1587286261: 

> Maybe even create a template for a template that would be shipped with copier?
> copier new_template that would create all expected files for a template?

@sisp you'll love the idea!

After the recent discovery of meta-templates, we can use that technique to provide a template hosted under copier-org to help template creators bootstrapping their new templates. 

I guess it makes sense that it should be a separate repo. Any ideas or suggestions for it?

**Comment by sisp** [comment]
Generally, yes, I just wonder how much there is to bootstrap for a new template project. :thinking: See also my [thought about omitting `{{ _copier_conf.answers_file }}.jinja`](https://github.com/copier-org/copier/issues/983#issuecomment-1447869262).

**Reply by yajo** [reply]
Yes, there is not much to bootstrap indeed. And it wouldn't need updates support. But having a base layout that acts as a self-documenting example could be good for newbies.
